nevergrad/optimization/oneshot.py

```diff
@@ -166,12 +204,15 @@ class SamplingSearch(base.ParametrizedFamily):
 
     # pylint: disable=unused-argument
     def __init__(self, *, sampler: str = "Halton", scrambled: bool = False, middle_point: bool = False,
-                 cauchy: bool = False, scale: float = 1., rescaled: bool = False) -> None:
+                 quasi_opposite: str = "none",
+                 cauchy: bool = False, supercalais: bool = False, scale: float = 1., rescaled: bool = False) -> None:
```

Comment:
is there no other better name that `supercalais`? :s 
+ update docstring

Reply:
Done, as well as other recommendations from you.